axelrod/result_set.py

```diff
@@ -45,6 +47,8 @@ def __init__(self, players, interactions, progress_bar=True):
             progress_bar : bool
                 Whether or not to create a progress bar which will be updated
         """
+        if game is not None:
+            self.game = game
```

Comment:
Which is what I should have suggested in the first place, but was too lazy to type out in full. sorry.